stories/timeline/props-table.tsx

```diff
@@ -102,11 +102,27 @@ const DATA: ApiTableSectionProps[] = [
     {
         name: "TimelineStatusProps",
         attributes: [
+            {
+                name: "",
+                description: (
+                    <>
+                        This component also inherits props from&nbsp;
+                        <a
+                            href="https://designsystem.life.gov.sg/?path=/docs/modules-pill--docs"
+                            rel="noreferrer"
+                            target="_blank"
+                        >
+                            PillProps
+                        </a>
+                    </>
+                ),
+            },
             {
                 name: "type",
                 mandatory: true,
-                description: "The color style of the pill",
-                propTypes: [`"dark"`, `"light"`],
+                description:
+                    "The style of the pill, `outline` `solid` are from PillType, `dark` `light` are retained for backward compatibility",
```

Comment:
would suggest to deprecate these properties and defer to the PillProps instead

```suggestion
                description: (
                    <>
                        The style of the pill
                        <br />
                        <br />
                        <strong>Note</strong>: <code>dark</code> and <code>light</code> are deprecated and will be removed in future versions
                    </>
                ),
```

for the label, we can add a deprecation notice as well